src/vs/workbench/parts/tasks/electron-browser/task.contribution.ts

```diff
@@ -239,6 +240,10 @@ class ConfigureTaskRunnerAction extends Action {
 					contentPromise = TPromise.as(selection.content);
 				}
 				return contentPromise.then(content => {
+					let editorConfig = this.configurationService.getConfiguration<any>();
+					if (editorConfig.insertSpaces) {
+						content = content.replace(/(\n)(\t+)/g, (_, s1, s2) => s1 + strings.repeat(' ', s2.length * editorConfig.tabSize));
```

Comment:
I can remove it, just want to be careful with the tabs which can be defined inside of the tasks. This combination fixes all the possible tabs in indentation right now defined in the vscode.